### ISSUE TITLE
GDPR

### ISSUE THREAD

**Issue by stieben** [peerhaven/peerhaven#14]
How does [GDPR](https://www.eugdpr.org/) impact what we want to do?

**Comment by stieben** [comment]
- [ ] Analytics
- [ ] Cookies
- [ ] Server logs

**Comment by stieben** [comment]
GDPR information:

- [GDPR Requirements in Plain English](https://blog.varonis.com/gdpr-requirements-list-in-plain-english/)
- [GDPR Hysteria](https://jacquesmattheij.com/gdpr-hysteria)
- [GDPR Hysteria, Part II: Nuts and Bolts, actionable advice](https://jacquesmattheij.com/gdpr-hysteria-part-ii-nuts-and-bolts)

**Comment by stieben** [comment]
[GitLab](https://about.gitlab.com/) has a fantastic cookie declaration.

**Comment by stieben** [comment]
I have to adjust my definition of fantastic because the [privacy settings screen of JTL](https://guide.jtl-software.de/jtl-wawi/) is fantastic.

**Comment by stieben** [comment]
- [ ] Create privacy policy:
  - [ ] What data is collected
  - [ ] How data is collected
  - [ ] How long data is kept
  - [ ] How data is used
  - [ ] Who has access to that data
- [ ] Save information about how and when consent is given by the user (humans of age 16+)
- [ ] Implement proper data life-cycle management (with options to export and delete personal data by user request)
- [ ] Provide a contact point and procedure for who to contact if violations are found (part of privacy policy)
- [ ] Store all data inside the EU
- [ ] If user data is changed by us, we need to tell them about it
- [ ] If we have a data breach, we need to notify both the authorities and the affected users within 72 hours

Examples of GDPR communication pages: [Hotjar](https://www.hotjar.com/legal/compliance/gdpr-commitment/) and [Facebook](https://www.facebook.com/business/gdpr)

The bottom line is that we will do our best to comply with the regulations and that acting in the user's interest is the heart of peerhaven, anyway. If despite our best intentions we manage to mess up, the maximum fine is 4% of our global turnover – which would be not too bad.

**Comment by stieben** [comment]
There are some interesting options for generating these documents online:

- [TermsFeed](https://www.termsfeed.com/) (free)
- [Termly](https://termly.io/) (free)
- [GetTerms](https://getterms.io/) ($49 once)
- [iubenda](https://www.iubenda.com/) (99€ per year and language)

We'll take care of these when we launch.
